.gemini/skills/extension-creator/references/introduction.md

```
@@ -1,377 +1,377 @@
+
+You can update all extensions with:
+
+```
+gemini extensions update --all
+```
+
+### Create a boilerplate extension
+
+We offer several example extensions `context`, `custom-commands`,
+`exclude-tools` and `mcp-server`. You can view these examples
+[here](https://github.com/google-gemini/gemini-cli/tree/main/packages/cli/src/commands/extensions/examples).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)